recipe/meta.yaml

```diff
@@ -1,37 +1,31 @@
 {% set name = "eth-hash" %}
-{% set version = "0.5.1" %}
-
+{% set version = "0.7.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9805075f653e114a31a99678e93b257fb4082337696f4eff7b4371fe65158409
+  sha256: bacdc705bfd85dadd055ecd35fd1b4f846b671add101427e089a4ca2e8db310a
 
 build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [py<39]
 
 requirements:
   host:
+    - python
     - pip
-    - python >=3.5,<4.0
-    - setuptools-markdown
-    - pycryptodome
+    - wheel
+    - setuptools
   run:
-    - python >=3.5,<4.0
-    - pycryptodome >=3.6.6,<4
-  run_constrained:
-    - pycryptodome >=3.6.6,<4
-    - pysha3 >=1.0.0,<2.0.0
+    - python
 
 test:
   imports:
     - eth_hash
-    - eth_hash.backends
```

Comment:
This is fine to check all `packages` (`backends` has` __init__.py`) inside the `eth_hash`, see https://github.com/ethereum/eth-hash/tree/v0.7.0/eth_hash